src/containers/AllSubjectsPage/SubjectLink.tsx

```diff
@@ -34,7 +34,6 @@ const StyledIconButton = styled(IconButtonV2)`
 `;
 
 const SubjectSafeLink = styled(SafeLink)`
-  ${fonts.sizes('18px', '24px')};
```

Comment:
Hvis vi først skal fjerne dette, skal vi da også fjerne font-weight?

Reply:
Mjaa, den har jo faktisk noe å si for utseendet her, lurer på om vi skal avvente til min ndla design er klart 🤔